packages/auth/src/AuthProvider.tsx

```diff
@@ -168,7 +172,13 @@ export const AuthProvider = (props: AuthProviderProps) => {
       props.config as SupportedAuthConfig
     )
 
-    setRwClient(client)
+    // This timeout lets default tests that only test that a component renders
+    // pass
+    setTimeout(() => {
```

Comment:
This is a bit hacky... But it does make it so we don't have to make any updates to already existing RW Apps out there.

If we want we could wrap this in `process.env.NODE_ENV === 'testing'` and skip the timeout in prod

Reply:
Thinking about this some more I really don't like the "wrap in env check" idea. If we had that we'd no longer be testing the same code as we run in prod.